dom/fetch/FetchDriver.cpp

```diff
@@ -808,9 +810,9 @@ nsresult FetchDriver::HttpFetch(
       MOZ_ASSERT(!mAltDataListener);
       mAltDataListener = new AlternativeDataStreamListener(
           this, chan, aPreferredAlternativeDataType);
-      rv = chan->AsyncOpen(mAltDataListener);
+      chanListener = mAltDataListener;
```

Comment:
I couldn't easily tell what "alternate data" in this context even was so I just opted to not record it. We can always add it if we find a case where it matters though.